modules/ztp-configuring-ptp-fast-events.adoc

```diff
@@ -56,21 +63,74 @@ You can configure PTP fast events for vRAN clusters that are deployed using the
 <5> Optional. If the `ptpClockThreshold` stanza is not present, default values are used for the `ptpClockThreshold` fields. The stanza shows default `ptpClockThreshold` values. The `ptpClockThreshold` values configure how long after the PTP master clock is disconnected before PTP events are triggered. `holdOverTimeout` is the time value in seconds before the PTP clock event state changes to `FREERUN` when the PTP master clock is disconnected. The `maxOffsetThreshold` and `minOffsetThreshold` settings configure offset values in nanoseconds that compare against the values for `CLOCK_REALTIME` (`phc2sys`) or master offset (`ptp4l`). When the `ptp4l` or `phc2sys` offset value is outside this range, the PTP clock state is set to `FREERUN`. When the offset value is within this range, the PTP clock state is set to `LOCKED`.
 
-. Apply the following `PolicyGenTemplate` changes to your specific site YAML files, for example, `example-sno-site.yaml`:
-
-.. In `.sourceFiles`, add the `Interconnect` CR file that configures the AMQ router to the `config-policy`:
+. Configure a `LocalVolume` (`LV`) to persist the PTP events subscription on the host.
++
+For example, add the following YAML to the `group-du-sno-ranGen.yaml` file:
 +
 [source,yaml]
 ----
-- fileName: AmqInstance.yaml
+- fileName: StorageLV.yaml <1>
   policyName: "config-policy"
+  spec:
+    storageClassDevices:
+      - storageClassName: "storage-class-http-events" <2>
+        volumeMode: Filesystem
+        fsType: xfs
+        devicePaths:
+          - /dev/disk/by-partlabel/httpevent1
+          - /dev/disk/by-partlabel/httpevent2
+- fileName: StoragePV.yaml  <3>
 ----
++
+<1> PTP events with HTTP transport requires local storage on the cluster node provided by the Local Storage Operator.
+<2> You need `storage-class-http-events` to use HTTP transport with the PTP Operator.
+Use the `ignitionConfigOverride` field in the related `SiteConfig` CR to manage the disk labels.
+<3> Optional. Configures storage for the `cloud-event-proxy` pubsubstore.
+
+. Configure the individual site `SiteConfig` CR that you use to provision the managed cluster. For example, in `example-sno-site-1.yaml`, add the following YAML (`ignitionConfigOverride`) to `.nodes`:
++
+[source,yaml]
+----
+spec:
+  ...
+  nodes:
+    - hostName: "example-node1.example.com"
+      role: "master"
+      ...
+      ignitionConfigOverride:
+        '{"ignition":{"version":"3.2.0"},"storage":{"disks":[{"device":"/dev/disk/by-id/wwn-0x11111000000asd123","wipeTable":false,"partitions":[{"sizeMiB":16,"label":"httpevent1","startMiB":350000},{"sizeMiB":16,"label":"httpevent2","startMiB":350016}]}],"filesystem":[{"device":"/dev/disk/by-partlabel/httpevent1","format":"xfs","wipeFilesystem":true},{"device":"/dev/disk/by-partlabel/httpevent2","format":"xfs","wipeFilesystem":true}]}}'
+----
+
+. Configure a `PersistentVolume` (`PV`) custom resource (CR) in the `StoragePV.yaml` file.
```

Comment:
This step can be removed since it is described in the Step 2 item 3. Of you can move the contents from Step 2 item 3 here and list it as optional.  The StoragePV.yaml file will be added in PR https://github.com/openshift-kni/cnf-features-deploy/pull/1440, so there is no need to describe its content of how to add the source file.